apps/fishing-map/features/workspace/activity/ActivityFilters.tsx

```diff
@@ -122,6 +122,7 @@ function ActivityFilters({ dataview: baseDataview }: ActivityFiltersProps): Reac
 
   const onDataviewFilterChange = useCallback(
     (dataviewInstance: UrlDataviewInstance) => {
+      console.log(newDataviewInstanceConfig)
```

Comment:
👀

Reply:
yes yes, I didn't add you as reviewer, it wasn't done